app/grandchallenge/products/models.py

```diff
@@ -132,7 +132,7 @@ class FDAStatus(models.TextChoices):
     countries = models.TextField()
     distribution = models.CharField(max_length=100, blank=True)
     institutes_research = models.TextField()
-    institutes_clinic = models.TextField()
+    institutes_clinical = models.TextField()
```

Comment:
As you changes the model in `models.py` you need to include a migration, you can do this with `make migrations`, run black, and then add and commit the generated file.

Reply:
@Kicksta This will be the cause of your local error

Reply:
Yup, changed it and then saw your comment ;) this was my way around it.